<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingsWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 1. 顶层窗口样式：浅色半透明，带圆角和淡边框 */
QWidget#SettingsWindow {
    background-color: rgba(255, 255, 255, 160); 
    border-radius: 15px;
    border: 1px solid rgba(0, 0, 0, 30);
}

/* 2. 文本标签样式：深灰文字，加粗更清晰 */
QLabel {
    background-color: transparent;
    color: #444444;
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 14px;
    font-weight: bold;
}

/* 3. 滑动条 (QSlider) 样式美化 */
QSlider::groove:horizontal {
    border: 1px solid #DDDDDD;
    height: 6px;
    background: rgba(0, 0, 0, 15); /* 槽体半透明 */
    margin: 2px 0;
    border-radius: 3px;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #66b3ff, stop:1 #0078d7); /* 蓝色渐变滑块 */
    border: 1px solid #005a9e;
    width: 16px;
    height: 16px;
    margin: -6px 0; /* 让滑块居中并稍大于槽体 */
    border-radius: 8px;
}

QSlider::handle:horizontal:hover {
    background: #44a0ff; /* 悬停时变亮 */
}

/* 4. 下拉选择框 (QComboBox) 样式 */
QComboBox {
    border: 1px solid #CCCCCC;
    border-radius: 4px;
    padding: 3px 10px;
    background: rgba(255, 255, 255, 200);
    color: #333333;
    min-width: 120px;
}

QComboBox:hover {
    border: 1px solid #0078d7;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: none;
}

/* 5. 按钮 (QPushButton) 样式 */
QPushButton {
    background-color: rgba(255, 255, 255, 220);
    border: 1px solid #CCCCCC;
    border-radius: 6px;
    padding: 6px 18px;
    color: #333333;
    font-weight: normal;
}

QPushButton:hover {
    background-color: #0078d7;
    color: white;
    border: 1px solid #005a9e;
}

QPushButton:pressed {
    background-color: #005a9e;
}

/* 6. 特殊处理：准心图片预览区域 */
QLabel#label_crosshair {
    background-color: rgba(0, 0, 0, 10); /* 给准心背后一个极淡的阴影，方便看清浅色准心 */
    border: 1px dashed rgba(0, 0, 0, 40);
    border-radius: 10px;
}</string>
  </property>
  <widget class="QLineEdit" name="sensitiveLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: transparent;
    border: none;
    color: white;
    font-size: 16px;
    padding-left: 12px;
}

QLineEdit:focus {
    outline: none;
}</string>
   </property>
   <property name="text">
    <string>灵敏度</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="crosshair">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: transparent;
    border: none;
    color: white;
    font-size: 16px;
    padding-left: 12px;
}

QLineEdit:focus {
    outline: none;
}</string>
   </property>
   <property name="text">
    <string>准心设置</string>
   </property>
  </widget>
  <widget class="QSlider" name="sensitiveSlider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>50</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_crosshair">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/crosshair.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditR">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: transparent;
    border: none;
    color: white;
    font-size: 16px;
    padding-left: 12px;
}

QLineEdit:focus {
    outline: none;
}</string>
   </property>
   <property name="text">
    <string>R</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditG">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: transparent;
    border: none;
    color: white;
    font-size: 16px;
    padding-left: 12px;
}

QLineEdit:focus {
    outline: none;
}</string>
   </property>
   <property name="text">
    <string>G</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: transparent;
    border: none;
    color: white;
    font-size: 16px;
    padding-left: 12px;
}

QLineEdit:focus {
    outline: none;
}</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditA">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: transparent;
    border: none;
    color: white;
    font-size: 16px;
    padding-left: 12px;
}

QLineEdit:focus {
    outline: none;
}</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_R">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TickPosition::NoTicks</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_G">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_B">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_A">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>11</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回启动器</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLang">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: transparent;
    border: none;
    color: white;
    font-size: 16px;
    padding-left: 12px;
}

QLineEdit:focus {
    outline: none;
}</string>
   </property>
   <property name="text">
    <string>语言设置</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Chinese-zh 简体中文</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>English 英语</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLang_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: transparent;
    border: none;
    color: white;
    font-size: 16px;
    padding-left: 12px;
}

QLineEdit:focus {
    outline: none;
}</string>
   </property>
   <property name="text">
    <string>分辨率</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="resolutionBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1080P 1920x1080</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>720P 1280x720</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
